CHANGELOG.md

```diff
@@ -3,7 +3,7 @@
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## v2.6.0dev - nfcore/mhcquant
+## v2.6.0dev - [date]
```

Comment:
dont you need to add the date here?

Reply:
I think its a placeholder, see e.g. https://github.com/nf-core/rnaseq/blob/dev/CHANGELOG.md#v3140dev---date